editor/editor_autoload_settings.h

```diff
@@ -74,9 +74,12 @@ class EditorAutoloadSettings : public VBoxContainer {
 	String selected_autoload;
 
 	Tree *tree;
-	EditorLineEditFileChooser *autoload_add_path;
+	HBoxContainer *autoload_add_path;
 	LineEdit *autoload_add_name;
 	Button *add_autoload;
+	LineEdit *line_edit;
+	Button *button;
```

Comment:
You should use more descriptive names than `line_edit` and `button` for class properties.

Reply:
Yeah. I agree. I couldn't of any for button without making it. I'll think of something.

Reply:
Can you advise me some names for button and line_edit? I thought of `autoload_add_path` for line_edit so that it matches with other LineEdit. Still not sure for button. I was thinking of `autoload_add_path` for button.

Reply:
Since the other one is called `add_autoload`, I think `choose_autoload_path` is fine.

Reply:
Check now.

Reply:
LGTM, thanks